integrations/deploy.yaml

```diff
@@ -43,8 +43,228 @@
   platform_info:
     group: 'include'
     distro: 'ubuntu'
   quick_start: -1
+- <<: *linux
+  id: deploy-debian
+  meta:
+    <<: *linux_meta
+    name: Debian
+    link: https://www.debian.org/
+    icon: 'debian.png'
+  platform_info:
+    group: 'include'
+    distro: 'debian'
+  quick_start: -1
+- <<: *linux
+  id: deploy-fedora
+  meta:
+    <<: *linux_meta
+    name: Fedora
+    link: https://www.fedoraproject.org/
+    icon: 'fedora.png'
+  platform_info:
+    group: 'include'
+    distro: 'fedora'
+  quick_start: -1
+- <<: *linux
+  id: deploy-rhel
+  meta:
+    <<: *linux_meta
+    name: Red Hat Enterprise Linux
+    link: https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux
+    icon: 'rhel.png'
+  platform_info:
+    group: 'include'
+    distro: 'rhel'
+  quick_start: -1
+- <<: *linux
+  id: deploy-almalinux
+  meta:
+    <<: *linux_meta
+    name: Alma Linux
+    link: https://almalinux.org/
+    icon: 'alma.png'
+  platform_info:
+    group: 'include'
+    distro: 'almalinux'
+  quick_start: -1
+- <<: *linux
+  id: deploy-alpinelinux
+  meta:
+    <<: *linux_meta
+    name: Alpine Linux
+    link: https://www.alpinelinux.org/
+    icon: 'alpine.png'
+  platform_info:
+    group: 'include'
+    distro: 'alpinelinux'
+  quick_start: -1
+- <<: *linux
+  id: deploy-amazonlinux
+  meta:
+    <<: *linux_meta
+    name: Amazon Linux
+    link: https://aws.amazon.com/amazon-linux-2/
+    icon: 'amazonlinux.png'
+  platform_info:
+    group: 'include'
+    distro: 'amazonlinux'
+  quick_start: -1
+- <<: *linux
+  id: deploy-archlinux
+  meta:
+    <<: *linux_meta
+    name: Arch Linux
+    link: https://archlinux.org/
+    icon: 'archlinux.png'
+  platform_info:
+    group: 'include'
+    distro: 'archlinux'
+  quick_start: -1
+- <<: *linux
+  id: deploy-centos
+  meta:
+    <<: *linux_meta
+    name: CentOS
+    link: https://www.centos.org/
+    icon: 'centos.png'
+  platform_info:
+    group: 'include'
+    distro: 'centos'
+  quick_start: -1
+- <<: *linux
+  id: deploy-manjarolinux
+  meta:
+    <<: *linux_meta
+    name: Manjaro Linux
+    link: https://manjaro.org/
+    icon: 'manjaro.png'
+  platform_info:
+    group: 'include'
+    distro: 'archlinux'
+  quick_start: -1
+- <<: *linux
+  id: deploy-oraclelinux
+  meta:
+    <<: *linux_meta
+    name: Oracle Linux
+    link: https://www.oracle.com/linux/
+    icon: 'oraclelinux.png'
+  platform_info:
+    group: 'include'
+    distro: 'oraclelinux'
+  quick_start: -1
+- <<: *linux
+  id: deploy-opensuse
+  meta:
+    <<: *linux_meta
+    name: SUSE Linux
+    link: https://www.suse.com/
+    icon: 'openSUSE.png'
+    group: 'include'
+    distro: 'opensuse'
+  quick_start: -1
+-
+  id: deploy-macos
+  meta:
+    name: MacOS
+    link: ''
+    categories:
+      - deploy.operating-systems
+    icon_filename: ''
+  keywords:
+    - MacOS
+    - mac
+  description: 'Run the following command on your OSX, macOS server to install and claim Netdata:"r'
```

Comment:
I saw in @Ferroin PR that this is only supported for Intel based Macs, if this is the case shouldn't we mention this here as well?